src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.TextTemplating/ScaffolderBuilderExtensions.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using Microsoft.DotNet.Scaffolding.TextTemplating;
+
+namespace Microsoft.DotNet.Scaffolding.Core.Builder;
+
+public static class ScaffolderBuilderExtensions
+{
+    public static IScaffoldBuilder WithTextTemplatingStep(
+        this IScaffoldBuilder builder,
+        string templateFilePath,
+        Type templateType,
+        string templateModelName,
+        object templateModel,
+        string outputPath)
+    {
+        return builder.WithStep<TextTemplatingStep>(config =>
```

Comment:
It doesn't seem like this is used? And it doesn't remove as much extra code as the others do. Consider cutting it?

Reply:
I think this is the only thing that wasn't changed/commented on? Do you think its important to leave it?

Reply:
removed yes